students/PeterB/Session 2/Grid.py

```diff
@@ -0,0 +1,32 @@
+plus = '+'
+minus = '- '
+bar = '|'
+space = ' '
+
+# print_grid2(3,4)Width = 4
+
+def print_row(length):
+    print(plus + space + minus * length + plus+space + minus * length + plus+space + minus * length+ plus)
```

Comment:
I can't help thinking there could be a more compact way to do this...

Reply:
But the bigger problem is that this will always makeit 3 cells wide -- we want that to adjustable, so you can make a 4x4 grid, for instance.